content/posts/towards-krustlet-v1-cncf.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Towards Krustlet 1.0 and CNCF Sandbox"
+description: "The good, bad, and ugly of using Rust after a year"
+date: 2021-07-25 00:00:00 +0000 UTC
+authorname: "Taylor Thomas and Radu Matei"
+authorlink: "https://github.com/deislabs"
+image: "images/twitter-card.png"
+tags: ["rust", "krustlet"]
+---
+
+Over a year ago, [we introduced Krustlet][intro], a [kubelet][kubelet]
+implementation that enables users to run both WebAssembly and traditional
+container workloads in the same Kubernetes cluster. If WebAssembly is new to
+you, check out the [section] below.
+
+We started the project with two clear goals in mind:
+
+- make it easy to deploy WebAssembly (and [WASI][wasi]) workloads on Kubernetes
+- contribute to the cloud native ecosystem by building core architectural pieces
+  of Kubernetes in [Rust][rust], a memory-safe systems programming langauge.
+
+Today we are extremely excited to announce that Krustlet was accepted in the
+[Cloud Native Computing Foundation (CNCF)][cncf] as a Sandbox project, alongside
+[Krator][krator], a Kubernetes Rust state machine operator framework written in
+Rust, as well as the first alpha release of Krustlet, [v1.0.0-alpha.1][alpha1].
+
+This marks the start of API and feature stability for Krustlet, and as we
+approach the final 1.0 release, of strong backwards compatibility guarantees.
+
+### The 1.0 roadmap
+
+Krustlet enables developers to build and run WebAssembly and WASI modules on
+Kubernetes. Today, you can take advantage of Kubernetes features such as pods,
+secrets, init containers, or volume mounts, and in the next couple of weeks, we
+will continue to stabilize the current API and feature set. Here are the
+specific guarantees for each type of release:
+
+- alpha: No backwards compatibility or API guarantees. However, we do not intend
+  to do any major breaking changes, but we reserve the right to should the need
+  arise
+- beta: Similar to alpha, we will not likely be breaking anything. The one
```

Comment:
I toss security fixes in here (and maybe in RC). I know that technically they are considered bugs, but these are the fixes that in helm have gotten the most violent feedback when breakages occur.